_data-prepper/pipelines/configuration/processors/csv.md

```diff
@@ -10,21 +10,33 @@ nav_order: 45
 
 ## Overview
 
-Takes in an Event and parses its CSV data into columns.
+The `csv` processor parses event data as comma-separated values (CSVs).
```

Comment:
`csv` processor converts CSV data to columns. You can find an example here. https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/csv-processor#example-with-user-specified-column-names-config

I think it should be something like this
The `csv` processor parses comma-separated values (CSVs) in event to columns.